src/Compilers/CSharp/Test/Emit3/Semantics/RecordTests.cs

```diff
@@ -3692,10 +3692,15 @@ public record B : A {
 }";
             var comp = CreateCompilationWithIL(new[] { source, IsExternalInitTypeDefinition }, ilSource: ilSource, parseOptions: TestOptions.Regular9);
             comp.VerifyEmitDiagnostics(
-                // (2,19): error CS8864: Records may only inherit from object or another record
+                // 0.cs(2,15): warning CS0114: 'B.EqualityContract' hides inherited member 'A.EqualityContract'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
                 // public record B : A {
-                Diagnostic(ErrorCode.ERR_BadRecordBase, "A").WithLocation(2, 19)
-                );
+                Diagnostic(ErrorCode.WRN_NewOrOverrideExpected, "B").WithArguments("B.EqualityContract", "A.EqualityContract").WithLocation(2, 15),
+                // 0.cs(2,15): warning CS0114: 'B.PrintMembers(StringBuilder)' hides inherited member 'A.PrintMembers(StringBuilder)'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
+                // public record B : A {
+                Diagnostic(ErrorCode.WRN_NewOrOverrideExpected, "B").WithArguments("B.PrintMembers(System.Text.StringBuilder)", "A.PrintMembers(System.Text.StringBuilder)").WithLocation(2, 15),
+                // 0.cs(2,19): error CS8864: Records may only inherit from object or another record
+                // public record B : A {
+                Diagnostic(ErrorCode.ERR_BadRecordBase, "A").WithLocation(2, 19));
```

Comment:
note: these tests are very esoteric.  they're validating the behavior if you compile against something that looks similar, but is not exaclty a record, and then derive from it.  so the extra warnings are not problematic.  this is not a scenario users will normally hit.